### PR TITLE
Linkreview

### PR DESCRIPTION
**Pull request by hgotar1**
Feedback to have more specific links for CSAs and customers
Recommendation 1: If you give a partner access to administer your tenant, use Azure Lighthouse.
Actual: https://learn.microsoft.com/en-gb/azure/cloud-adoption-framework/ready/landing-zone/design-area/azure-billing-microsoft-customer-agreement#design-recommendations 
More specific: https://learn.microsoft.com/en-us/azure/lighthouse/overview 

Recommendation 2: Use of Enterprise Dev/Test Subscriptions to reduce costs for non-production workloads.
Actual: https://learn.microsoft.com/en-gb/azure/cloud-adoption-framework/ready/landing-zone/design-area/azure-billing-enterprise-agreement#design-considerations 
More specific: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/landing-zone/design-area/management-application-environments 

Recommendation 3: Use managed identities instead of service principals for authentication to Azure services. You can check for existing service principals via Entra ID > Sign in Logs > Service principal logins.
Actual: https://learn.microsoft.com/en-gb/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access-landing-zones#identity-and-access-management-in-the-azure-landing-zone-accelerator 
More specific: https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview

### CODE REVIEW

File: checklists/alz_checklist.en.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
             "guid": "5cf9f485-2784-49b3-9824-75d9b8bdb57b",
             "id": "A03.05",
             "severity": "Low",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/azure-billing-enterprise-agreement#design-considerations",
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/management-application-environments",
```
[the remark this answers is not in the document]
Comment (igorjnzl) [reply checklists/alz_checklist.en.json:112]: @hgotar1 would this link be better? 

https://azure.microsoft.com/en-au/pricing/offers/ms-azr-0148p

[the remark this answers is not in the document]
Comment (brsteph) [reply checklists/alz_checklist.en.json:112]: Yeah, I think so.  Making it region agnostic as https://azure.microsoft.com/pricing/offers/ms-azr-0148p would be a good landing page!

[the remark this answers is not in the document]
Comment (brsteph) [reply checklists/alz_checklist.en.json:112]: Also, feel free to merge any time on this.

[the remark this answers is not in the document]
Comment (hgotar1) [reply checklists/alz_checklist.en.json]: link has been updated as per comment request

**Pull request merged**